vendor/k8s.io/kubernetes/cmd/kube-controller-manager/app/patch_imagetemplate.go

```diff
@@ -113,25 +115,27 @@ func ExpandStrict(s string, fns ...KeyFunc) (string, error) {
 type KeyFunc func(key string) (string, bool)
 
 // Versions is a KeyFunc for retrieving information about the current version.
-func Versions(key string) (string, bool) {
```

Comment:
I expected to see one line added here for `overrideVersion=GetOpenshiftVersion()`